### PR TITLE
Prepare jiralert for Criteo usage

### PR DESCRIPTION
**Pull request by Thib17**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -3,56 +3,39 @@
...
-----
-{% endfor %}
-
-
-alert_group_key={{ groupKey }}
-''')
+JINJA_ENV = jinja2.Environment(loader=jinja2.FileSystemLoader(ROOT_DIR))
+JINJA_ENV.filters['prepareGroupKey'] = prepareGroupKey
 
+summary_tmpl = JINJA_ENV.get_template('templates/summary')
```
Comment (iksaif) [inline main.py]: summary.tmpl instead of just summary ?

File: main.py [hunk 2]
```diff
@@ -3,56 +3,39 @@
...
+description_tmpl = JINJA_ENV.get_template('templates/description')
 description_boundary = '_-- Alertmanager -- [only edit above]_'
 
 # Order for the search query is important for the query performance. It relies
 # on the 'alert_group_key' field in the description that must not be modified.
-search_query = 'project = %s and labels = "alert" and description ~ "alert_group_key=%s"'
+search_query = 'project = %s and labels = "jiralerts" and status not in (%s) and description ~ "alert_group_key=%s"'
```
Comment (iksaif) [inline main.py]: why do we need to change the label ?

File: main.py [hunk 3]
```diff
@@ -149,14 +118,21 @@ def metrics():
 @click.command()
 @click.option('--host', help='Host listen address')
 @click.option('--port', '-p', default=9050, help='Listen port for the webhook')
+@click.option('--res_trans', default="resolve issue,close issue",
```
Comment (iksaif) [inline main.py]: /trans/transitions/

**Review by iksaif — CHANGES_REQUESTED** [review]

**Review by iksaif — APPROVED** [review]

**Comment by InformatiQ** [comment]
LGTM

**Pull request merged**